src/AddIns/Uno.UI.Lottie/ts/Uno.UI.Lottie.ts

```diff
@@ -132,7 +141,7 @@ namespace Uno.UI {
 		}
 
 		private static createAnimation(properties: LottieAnimationProperties): RunningLottieAnimation {
-			var existingAnimation = this._runningAnimations[properties.elementId];
+			const existingAnimation = this._runningAnimations[properties.elementId];
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the parent class name instead of `this` when in a `static` context.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5436651)